lxd/instance_file.go

```diff
@@ -390,23 +390,29 @@ func instanceFileHead(s *state.State, inst instance.Instance, path string, r *ht
 //	    description: Raw file content
 //	  - in: header
 //	    name: X-LXD-uid
-//	    description: File owner UID
+//	    description: Base 10 integer for the file owner UID
 //	    schema:
 //	      type: integer
 //	    example: 1000
 //	  - in: header
 //	    name: X-LXD-gid
-//	    description: File owner GID
+//	    description: Base 10 integer for the file owner GID
```

Comment:
Can you split this commit into logical changes:

- Comments and security/validation check improvements.
- Changes specific to the new API extension.

So that we can backport the former and not the latter into the LTS series.

Reply:
It would make sense to me to include shared.ParseLXDFileHeaders change here in a separate commit too, and then have additional commits that update add the new field and shared.ParseLXDFileHeaders to handle the new header and use the new field here.